src/components/modals/AddQuestion/AddQuestion.jsx

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Where did you need to spread? Which line is causing this?

Reply:
React Dropzone library (file drag and drop) requires prop spreading

Reply:
Since it is one line let's add an ignore for that line only.